packages/vuetify/src/components/VDataTable/VDataTable.ts

```diff
@@ -149,7 +149,12 @@ export default VDataIterator.extend({
     },
     computedItemsPerPage (): number {
       const itemsPerPage = this.options && this.options.itemsPerPage ? this.options.itemsPerPage : this.itemsPerPage
-      if (this.sanitizedFooterProps.itemsPerPageOptions && !this.sanitizedFooterProps.itemsPerPageOptions.includes(itemsPerPage)) {
+      if (
+        this.sanitizedFooterProps.itemsPerPageOptions &&
```

Comment:
I know this is nitpicky but this is a very long variable name with a very long property name. Could we not extract the property to make it a bit more concise?

```
const { itemsPerPageOptions } = this.sanitizedFooterProps
```